node/Cargo.toml

```diff
@@ -18,12 +18,16 @@ name = "data-avail"
 
 [dependencies]
 # Internals
-da-primitives= { path = "../primitives/avail" }
+da-primitives = { git = "https://github.com/maticnetwork/avail-core.git", tag = "kate/v0.2.1", default-features = false }
```

Comment:
Maybe idea for the future, it would be great if we could split primitevies to substrate related ones, (like block header), and avail related once, like matrix data index. Former would be in avail repo, while latter in avail-core, ready for sharing and testing. WDYT?

Reply:
Good idea. I've created a [new ticket](https://linear.app/matic/issue/AVA-396/split-substrate-primitives-and-avail-related-types-on-avail-core) for that.